specs/core/0_beacon-chain.md

```diff
@@ -663,13 +662,13 @@ full_deposit_count: uint256
 @payable
 @public
 def deposit(deposit_input: bytes[2048]):
-    assert msg.value >= as_wei_value(MIN_DEPOSIT, "ether")
-    assert msg.value <= as_wei_value(MAX_DEPOSIT, "ether")
+    assert msg.value >= MIN_DEPOSIT_AMOUNT
```

Comment:
Do you want `as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")` here?

Reply:
Fixed